scranShipper/src/RestaurantMenu/index.jsx

```diff
@@ -0,0 +1,46 @@
+import {useEffect, useState} from "react";
+import './style.css'
+const RestaurantMenu = (props) => {
+
+    console.log(`this is the rest prop from restuarant menu ${props.restaurantId}`);
+    const [menus, getMenu] = useState([]);
+    const fetchMenus = (id) => {
+        fetch('https://food-delivery-api.dev.io-academy.uk/restaurants/' + id).then(response => {
+            return response.json();
+        }).then((data) => {
+            getMenu(data.foodItems);
+        })
+    }
+
+    useEffect(() => {
+        fetchMenus(props.restaurantId)
+    }, []);
+
+    const displayMenu = (item) => {
+    return (
+        <div className='menuItem'>
+            <h3 className='name'>{item.foodName}</h3>
+            <div className='info'>
+            <p className='calories'>Calories: {item.calories}</p>
```

Comment:
You are conditionally rendering the other food info, but not calories, why is that?